packages/flarevm.installer.vm/tools/chocolateyinstall.ps1

```diff
@@ -76,7 +76,10 @@ try {
     # Display installer log if available
     $logPath = Join-Path ${Env:VM_COMMON_DIR} "log.txt"
     if ((Test-Path $logPath)) {
-        Write-Host "[-] Please see installer log for any errors: $logPath" -ForegroundColor Yellow
+        Write-Host "[-] Please check the following logs for any errors:" -ForegroundColor Yellow
```

Comment:
This is written to `C:\ProgramData\_VM\log.txt`, right? I would add all the logs we mention there to the artifacts of the actions and display `C:\ProgramData\_VM\log.txt` in the action log. Related to https://github.com/mandiant/VM-Packages/pull/117

Reply:
This message is being displayed to the user in the PowerShell Window. We could also add this message to the `log.txt` file.

Reply:
@Ana06  Also add the following logs to the artifacts: `%PROGRAMDATA%\chocolatey\lib-bad\*\install_log.txt`

Packages may create a per-install log to use for whatever they wish (see: https://github.com/mandiant/VM-Packages/blob/f27bacad8c5e658fec4434a5f1f57122ebcb7d94/packages/common.vm/tools/vm.common/vm.common.psm1#L169). The packages `libraries.python*.vm` use this to redirect output from pip when installing Python modules. It'd be good to grab these per-install log files as well. You'll likely need to use a regex to find them, but they should be in a subdirectory within `lib-bad` somewhere.

Reply:
Documented in https://github.com/mandiant/VM-Packages/issues/133